tools/mobdbconverter.py

```diff
@@ -260,7 +260,7 @@ def readItemDB(inFile, itemDb):
                 elif line[:3] == "Id:":
                     try:
                         itemId = int(line[4:])
-                    except:
+                    except BaseException:
```

Comment:
This will warn as 'Too broad clause because of no exception class specified.
My suggestion is:
`except Exception as e:`
   `print(e)`
So we can know which exception will be raised.

Reply:
i think try/except here present for avoid int conversion error aka ``ValueError`` exception
Exception or BaseException hide too many possible issues